database/changesets/20210422_TerraExecutorDetailsTable.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="chengche@broadinstitute.org"
+               dbms="postgresql"
+               id="20210422"
+               logicalFilePath="20210422_TerraExecutorDetailsTable.xml"
+               runInTransaction="false">
+        <comment>
+            the details table of 'Terra' executor
+        </comment>
+        <sql>
+            CREATE TYPE TerraDetails AS (
+                id                     bigint,      -- unique to this table
+                rawls_submission_id    text,        -- id of the Rawls submission
+                workflow_id            text,        -- id of the Terra workflow
```

Comment:
(Asking selfishly because it will help me with my code) how do we link a `workflow_id` back to a fully-qualified workflow name?